src/snowflake/sqlalchemy/util.py

```diff
@@ -85,3 +94,15 @@ def _set_connection_interpolate_empty_sequences(
     else:
         # _dbapi_connection is a raw SnowflakeConnection
         dbapi_connection._interpolate_empty_sequences = flag
+
+
+def _update_connection_application_name(
+    **conn_kwargs: Dict[str, Any]
```

Comment:
I think `Any` is enough here according to https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values
```suggestion
    **conn_kwargs: Any
```

Reply:
updated